asciidoc/modules/con_mg_miscellaneous-changes-in-vertx.adoc

```diff
@@ -0,0 +1,60 @@
+[id="miscellaneous-changes-in-vertx_{context}"]
+= Miscellaneous changes in {VertX}
+
+The following section describes miscellaneous changes in {VertX} {4}.
+
+== Removed the `Starter` class
+
+The `Starter` class has been removed. Use the `Launcher` class instead to start your {VertX} applications without the `main()` method.
+
+== Isolated deployment for Java 8
+
+{VertX} {4} supports Java 11. This Java version does not support isolated class loading.
+In {VertX} {4}, isolated class loading will be supported for Java 8. However, the results may not be as expected.
+
+== Removed hook methods from context
+
+The methods `Context.addCloseHook()` and `Context.removeCloseHook()` methods have been removed from the `Context` class.
+
+== Removed the clone methods from options
+
+The methods `KeyCertOptions.clone()`, `TrustOptions.clone()`, and `SSLEngineOptions.clone()` have been removed. Use the methods `KeyCertOptions.copy()`, `TrustOptions.copy()`, and `SSLEngineOptions.copy()` instead.
+
+== Removed equals and hashcode methods from options
+
+The `VertxOptions.equals()` and `VertxOptions.hashCode()` methods have been removed.
```

Comment:
@vietj Can we add the reason why these methods were removed?There were no deprecations in Vert.x 3.x releases.

Reply:
They have been removed because they are useless and applications should not rely on it. So we removed them to have less maintenance.

Reply:
@vietj Ok.. No updates required to the existing note